static/js/src/contact-forms.js

```diff
@@ -1,49 +1,84 @@
-    if (event.currentTarget.checked) {
-        checkboxes[0].removeAttribute("required");
-    }
+  const checkboxes = Array.from(
+    document
+      .getElementById(fieldsetId)
+      .querySelectorAll("input[class='p-checkbox__input']")
+  );
+  if (event.currentTarget.checked) {
+    checkboxes[0].removeAttribute("required");
+  }
 }
 
 function getRadioItemValue(fieldset) {
-    const selectedRadio = fieldset.querySelector("input[name='how-many-machines-do-you-have']:checked")
-    return selectedRadio ? selectedRadio.value : ''
+  const selectedRadio = fieldset.querySelector(
+    "input[name='how-many-machines-do-you-have']:checked"
+  );
+  return selectedRadio ? selectedRadio.value : "";
 }
 
 function getCheckboxItemsAsCSV(fieldset) {
-    const checkboxes = Array.from(fieldset.querySelectorAll("input[class='p-checkbox__input']"));
-    return checkboxes.filter(item => item.checked).map(item => item.value).join(", ");
+  const checkboxes = Array.from(
+    fieldset.querySelectorAll("input[class='p-checkbox__input']")
+  );
+  return checkboxes
+    .filter((item) => item.checked)
+    .map((item) => item.value)
+    .join(", ");
 }
 
 function getCustomFields(event) {
-    // If you use Ubuntu, which version(s) are you using?
-    const ubuntuVersionsFieldset = document.getElementById("ubuntu-versions");
-    // What kind of device are you using?
-    const deviceUsedFieldset = document.getElementById("kind-of-device");
-    // How many machines?
-    const howManyMachinesFieldset = document.getElementById("how-many-machines");
-    // How do you consume open source?
-    const consumeOpenSourceFieldset = document.getElementById("how-do-you-consume-open-source");
-    // Do you have specific compliance or hardening requirements?
-    const hardeningRequirementsFieldset = document.getElementById("hardening-requirements");
-    // Who is responsible for tracking, testing and applying CVE patches in a timely manner?
-    const responsibleForTracking = document.getElementById("responsible-for-tracking");
+  event.preventDefault();
+  // If you use Ubuntu, which version(s) are you using?
+  const ubuntuVersionsFieldset = document.getElementById("ubuntu-versions");
+  // What kind of device are you using?
+  const deviceUsedFieldset = document.getElementById("kind-of-device");
+  // How many machines?
+  const howManyMachinesFieldset = document.getElementById("how-many-machines");
+  // How do you consume open source?
+  const consumeOpenSourceFieldset = document.getElementById(
+    "how-do-you-consume-open-source"
+  );
+  // Do you have specific compliance or hardening requirements?
+  const hardeningRequirementsFieldset = document.getElementById(
+    "hardening-requirements"
+  );
+  // Who is responsible for tracking, testing and applying CVE patches in a timely manner?
+  const responsibleForTracking = document.getElementById(
+    "responsible-for-tracking"
+  );
```

Comment:
This is likely why only form fields with ID's are being sent.

I don't have full context on this work, so please let me know if I am wrong. But I think we can automate this by looping through each field set and extracting the heading and input/textareas.